File: Baekjoon_Online_Judge/NO_01157/1157.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 09                           | #
# | Day 02                             | #
# | 단어 공부                           | #
# -------------------------------------- #


if __name__ == "__main__":
    string = r_input().rstrip().upper()

    counting = {}

    for ch in string:
        if ch not in counting:
            counting[ch] = 0
        
        counting[ch] += 1

    max_val = 0
    res = ''

    for alpha, val in counting.items():
        if val > max_val:
            max_val = val
            res = alpha
        
        elif val == max_val:
            res = '?'

    print(res)

File: Baekjoon_Online_Judge/NO_01157/1157.cpp
#include <iostream>
using namespace std;
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 09                           | 
 | Day 02                             | 
 | 단어 공부                          | 
 -------------------------------------- 
*/

int main(){
    string s;
    int save[26] = {0,};

    cin >> s;
    
    for (auto ch: s){
        if (ch >= 'a') ch -= 32;

        save[ch - 'A'] ++;
    }

    int max_cnt = 0;
    char res;

    for (int i = 0; i < 26; i ++){
        if (save[i] > max_cnt){
            max_cnt = save[i];
            res = i + 'A';
        }

        else if (save[i] == max_cnt) res = '?';
    }

    cout << res;

    return 0;
}